### ISSUE TITLE
CachedAuth created on each request

### ISSUE THREAD

**Issue by g4s8** [artipie/artipie#442]
`CachedAuth` instance is created on each request and schedules new task for cache cleanup. It should wrap origin auth object only once and start only one thread for cleanup

**Comment by 0pdd** [comment]
@g4s8 the puzzle [#460](https://github.com/artipie/artipie/issues/460) is still not solved.

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#442` is not assigned, can't get performer

**Comment by 0pdd** [comment]
@g4s8 the only puzzle [#460](https://github.com/artipie/artipie/issues/460) is solved here.
